frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -166,6 +168,11 @@ impl<T: Config> Pallet<T> {
 			maximum_allowed_voters,
 		);
 		let compact = Self::trim_compact(maximum_allowed_voters, compact, &voter_index)?;
+		let compact = Self::trim_compact_for_length(
```

Comment:
```suggestion
		let compact = Self::trim_compact_length(
```

wdyt about the naming?

Reply:
Yeah, works for me. I'll make that change now.

Reply:
https://github.com/paritytech/substrate/pull/8317/commits/909309c25fbf8f8780acbee47c13d6da80c9a118